.gitignore

```diff
@@ -3,4 +3,5 @@ build
 composer.lock
 phpunit.xml
 vendor
-reports
\ No newline at end of file
+reports
+.idea
```

Comment:
sigh, why do people always want to add it?
`.idea` shall be part of your own global `.gitignore`; not our responsibility.